From 0a2e0141add40230e8783c060b009a8a6ce0ad47 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Mon, 13 Apr 2020 02:11:13 +0300
Subject: [PATCH 132/133] tuning Amoled and GrayDim themes

	modified:   packages/overlays/AmoledDarkUIOverlay/res/values-night/colors.xml
	modified:   packages/overlays/GrayDimDarkUIOverlay/res/values-night/colors.xml
---
 .../res/values-night/colors.xml                    |  4 ++--
 .../res/values-night/colors.xml                    | 14 +++++++-------
 2 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/packages/overlays/AmoledDarkUIOverlay/res/values-night/colors.xml b/packages/overlays/AmoledDarkUIOverlay/res/values-night/colors.xml
index 052fb3896b4..e8796941ba7 100644
--- a/packages/overlays/AmoledDarkUIOverlay/res/values-night/colors.xml
+++ b/packages/overlays/AmoledDarkUIOverlay/res/values-night/colors.xml
@@ -30,8 +30,8 @@
     <color name="secondary_material_settings">#ff0f1012</color> <!-- material_grey_800 -->
     <color name="tertiary_material_settings">#ff151618</color> <!-- material_grey_600 -->
     <color name="quaternary_material_settings">#ff202126</color> <!-- material_grey_200 -->
-    <color name="material_grey_900">@android:color/black</color>
-    <color name="material_grey_850">@android:color/black</color>
+    <color name="material_grey_900">#ff0f1012</color>
+    <color name="material_grey_850">#ff0f1012</color>
     <color name="material_grey_800">#ff0f1012</color>
     <color name="material_grey_600">#ff151618</color>
     <color name="material_grey_300">#ff1a1b1e</color>
diff --git a/packages/overlays/GrayDimDarkUIOverlay/res/values-night/colors.xml b/packages/overlays/GrayDimDarkUIOverlay/res/values-night/colors.xml
index 17ef27a402a..b8bc42ae542 100644
--- a/packages/overlays/GrayDimDarkUIOverlay/res/values-night/colors.xml
+++ b/packages/overlays/GrayDimDarkUIOverlay/res/values-night/colors.xml
@@ -20,19 +20,19 @@
     <color name="notification_primary_text_color_dark">#ffe5e5e5</color>
     <color name="notification_secondary_text_color_dark">#b3e5e5e5</color>
     <color name="notification_default_color_dark">#ffe5e5e5</color>
-    <color name="notification_material_background_color">#ff2a2a2a</color> <!-- material_grey_900 -->
+    <color name="notification_material_background_color">#ff27282b</color> <!-- material_grey_900 -->
     <color name="primary_material_dark">#ff202124</color> <!-- material_grey_900 -->
     <color name="primary_material_settings">#ff202124</color> <!-- material_grey_900 -->
-    <color name="primary_dark_material_dark">#ff2a2a2a</color>
-    <color name="primary_dark_material_settings">#ff2a2a2a</color> <!-- material_grey_850 -->
-    <color name="primary_dark_device_default_dark">#ff2a2a2a</color> <!-- material_grey_850 -->
+    <color name="primary_dark_material_dark">#ff27282b</color>
+    <color name="primary_dark_material_settings">#ff27282b</color> <!-- material_grey_850 -->
+    <color name="primary_dark_device_default_dark">#ff27282b</color> <!-- material_grey_850 -->
     <color name="primary_dark_device_default_settings">#ff202124</color> <!-- main settings bg -->
-    <color name="background_floating_device_default_dark">#ff2a2a2a</color>
+    <color name="background_floating_device_default_dark">#ff27282b</color>
     <color name="secondary_material_settings">#ff303133</color> <!-- material_grey_800 -->
     <color name="tertiary_material_settings">#ff3a3a3a</color> <!-- material_grey_600 -->
     <color name="quaternary_material_settings">#ffdadce0</color> <!-- material_grey_200 -->
-    <color name="material_grey_900">#ff2a2a2a</color>
-    <color name="material_grey_850">#ff2a2a2a</color>
+    <color name="material_grey_900">#ff27282b</color>
+    <color name="material_grey_850">#ff27282b</color>
     <color name="material_grey_800">#ff303133</color>
     <color name="material_grey_600">#ff3a3a3a</color>
     <color name="material_grey_300">#ff3c4043</color>
-- 
2.17.1

